de.dashup.shared/src/main/java/de/dashup/shared/DatabaseModels/DatabaseWidget.java

```diff
@@ -0,0 +1,91 @@
+package de.dashup.shared.DatabaseModels;
+
+import com.google.gson.annotations.SerializedName;
+import de.dashup.shared.Enums.Size;
+
+public class DatabaseWidget extends DatabaseObject{
+
+    private Integer id;
+    private String name;
+    private String description;
+    @SerializedName("overview_text")
+    private String overviewText;
+    private String category;
+    @SerializedName("publisher_id")
+    private Integer publisherID;
+    @SerializedName("publication_date")
+    private Integer publicationDate;
```

Comment:
Why not using an `int`?

Reply:
Then you can use it as keys in data structures like the HashMap.